sleap/gui/commands.py

```diff
@@ -2029,6 +2033,34 @@ def ask(context: CommandContext, params: dict) -> bool:
 
         return len(filename) > 0
 
+class LinkVideoToSession(EditCommand):
+    topics = [UpdateTopic.video]
+
+    @staticmethod
+    def do_action(context: CommandContext, params: dict):
+        video = context.state["video"]
+        session = context.state["session"]
+
+        if video is None:
+            raise ValueError("No video selected.")
+
+        if session is None:
+            raise ValueError("No session selected.")
```

Comment:
This is okay, but also, we should never get the `ValueError` because we are disabling the `video_link_button` when there is no selected `RecordingSession` or no selected `Video`. And, we also need a selected `Camcorder`.